golang/dbos-toolbox/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/dbos-inc/dbos-transact-go/dbos"
+)
+
+/*****************************/
+/**** WORKFLOWS AND STEPS
+/*****************************/
+var (
+	wf = dbos.WithWorkflow(workflow)
+)
+
+func workflow(ctx context.Context, _ string) (string, error) {
```

Comment:
Why all these unused string inputs and outputs? Workflows must have them, no matter what?

Reply:
Yes, there are no optional parameters. There's a few workaround we can chat about.